content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -221,71 +221,113 @@ sysctl --system
+echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
+
+curl -L https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/Release.key | apt-key add -
+curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/Release.key | apt-key add -
+
+apt-get update
+apt-get install cri-o cri-o-runc
 ```
 {{% /tab %}}
 
-{{% tab name="CentOS/RHEL 7.4+" %}}
+{{% tab name="CentOS" %}}
 
-```shell
-# Install prerequisites
-curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/CentOS_7/devel:kubic:libcontainers:stable.repo
-curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}/CentOS_7/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}.repo
-```
+To install on the following operating systems, set the environment variable $OS as the appropriate field in the following table:
 
+| Operating system | $OS               |
+| ---------------- | ----------------- |
+| Centos 8         | `CentOS_8`        |
+| Centos 8 Stream  | `CentOS_8_Stream` |
+| Centos 7         | `CentOS_7`        |
+
+<br />
+Then, set `$VERSION` to be the cri-o version matching your kubernetes version.
+For instance, if you want to install cri-o 1.18, set `VERSION=1.18`.
+You can pin your installation to a specific release.
+To install version 1.18.3, set `VERSION=1.18:1.18.3`.
+<br />
+
+Then run
 ```shell
-# Install CRI-O
-yum install -y cri-o
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/devel:kubic:libcontainers:stable.repo
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/devel:kubic:libcontainers:stable:cri-o:$VERSION.repo
+yum install cri-o
 ```
+
 {{% /tab %}}
 
 {{% tab name="openSUSE Tumbleweed" %}}
 
 ```shell
 sudo zypper install cri-o
 ```
+{{% /tab %}}
+{{% tab name="Fedora" %}}
+
+Set `$VERSION` to be the cri-o version matching your kubernetes version.                                                                                                                                                                
+For instance, if you want to install cri-o 1.18, `VERSION=1.18`                                                                                                                                                                               
+You can find available versions with:
+```shell
+dnf module list cri-o
+```
+We do not support pinning to specific releases on Fedora.
```

Comment:
Avoid using ["we"](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we).
```suggestion
CRI-O does not support pinning to specific releases on Fedora.
```